internal/cmd/local/local_install.go

```diff
@@ -39,6 +41,17 @@ func (i *InstallCmd) Run(ctx context.Context, provider k8s.Provider, telClient t
 		return fmt.Errorf("unable to determine docker installation status: %w", err)
 	}
 
+	extraVolumeMounts, err := parseVolumeMounts(i.Volume)
+	if err != nil {
+		return err
+	}
+
+	for _, host := range i.Host {
+		if regexp.MustCompile(`\d+\.\d+\.\d+\.\d+`).MatchString(host) {
```

Comment:
We should replace this with the [net.ParseIP](https://pkg.go.dev/net#ParseIP) method, as that also supports ipv6 ip addresses.

Reply:
Nice, great idea